advanced/readme_kiali.adoc

```diff
@@ -0,0 +1,196 @@
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (usually,
+"kiali-istio-system.127.0.0.1.nip.io"), so let's access it:
+
+image::https://i.imgur.com/rG5tdZu.png[Login Page]
+
+The default credentials are "admin/admin", but it's recommended to change them
+before using it in production.
+
+== Generating Sample Data
+
+To show the capabilities of Kiali, you'll need an Istio-enabled application to
+be running. On the Kiali repository there's a sample application called
+Bookinfo that you can use to show these capabilities, and you can deploy it
+like this:
+
+[source, bash]
+----
+git clone git@github.com:kiali/kiali
+cd kiali
+./hack/istio/install-bookinfo-demo.sh
+----
+
+This will install the demo. Now we just need to visit it to generate data. We
+can get the URL like this:
+
+[source, bash]
+----
+oc project bookinfo
+oc get routes
+----
+
+From there, we just visit the "productpage" service and this will connect to
+all the other services on the demo.
+
+== Service Graph
+
+After you login, you should see the Service Graph page:
+
+image::https://i.imgur.com/PtKVfe4.png[Service Graph Page]
+
+It shows a graph with all the microservices, connected by the requests going
+through then. On this page, you can see how the services interact with each
+other.
+
+== Service Listing
+
+Click the Services link in the left navigation. On the Services page you can
+view a listing of all the services that are running in the cluster, and
+additional information about them, such as health status.
+
+image::https://i.imgur.com/Gytp5Wm.png[Service Listing Page]
+
+Click on a service name to view the Service Details page, where you can view
+additional information about each service.
+
+image::https://i.imgur.com/msQjCb4.png[Service Details]
+
+You can see the health status of the service (a Healthy service means one that
+is is up and responding correctly):
+
+image::https://i.imgur.com/MVpqDx8.png[Health Status]
+
+You can also see the deployments:
+
+image::https://i.imgur.com/TdyjCOu.png[Deployments]
+
+The source services:
+
+image::https://i.imgur.com/Qof6a8s.png[Source Services]
+
+And the virtual services on the Istio infrastructure:
+
+image::https://i.imgur.com/mlZknRv.png[Virtual Services]
+
+== Istio Config
+
+Click the Istio Config link on the left navigation. On this page, you can see
+all currently running config rules, such as Virtual Services, Route Rules,
+Routes, Circuit Breakers, Fault Injection and so on.
+
+image::https://i.imgur.com/KWhZ29z.png[Istio Config Page]
+
+== Distributed Tracing
+
+Click on the Distributed Tracing link in the left navigation. On this page you
```

Comment:
It gives me an error "Could not fetch Jaeger info Error: [ service: services "jaeger-query" not found ]" - On istio 0.8 is has changed to tracing-istio-system